README.md

```diff
@@ -20,22 +21,86 @@ To load in Visual Studio, from the Visual Studio File menu select Open > Project
 
 Optionally, to build in a Docker container, it is recommended to use Mono and xbuild.
 
+## Methods
+
+### Client
+`Client Client(uri, account)`
+- Create new Conjur instance
+   - uri - URI of the Conjur server. Example: https://myconjur.org.com/api
+   - account - Name of the Conjur account
```

Comment:
Github is weird with that. Every time I try to do backticks inline the format gets all messed up and the whole line is normally in a code block and always various other formatting issues

Reply:
@JakeQuilty it should be ok if you use single backticks as Srdjan suggested

if you're using code blocks (triple backticks) in a bulleted list, you just have to be careful about indentation, otherwise github does get confused